bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2926,6 +2934,15 @@ void ReplicaImp::onSeqNumIsStable(SeqNum newStableSeqNum, bool hasStateInformati
     IControlHandler::instance()->onStableCheckpoint();
   }
 }
+void ReplicaImp::sendRepilcaRestartReady() {
+  auto seq_num_to_stop_at = ControlStateManager::instance().getCheckpointToStopAt();
+  if (seq_num_to_stop_at.has_value()) {
+    unique_ptr<ReplicaRestartReadyMsg> readytToRestartMsg(
+        ReplicaRestartReadyMsg::create(config_.getreplicaId(), seq_num_to_stop_at.value()));
```

Comment:
In the future, when we will have an epoch value, it should be also part of this message.
Can you add a comment for this?